src/test/fixtures/composes.tsx

```diff
@@ -386,45 +580,67 @@ export const mockClones = (composeId) => {
           },
         },
       ],
+      meta: {
+        count: 4,
+      },
+      links: {
+        first: '',
+        last: '',
+      }
+    };
+  } else {
+    return {
+      data: [],
+      meta: {
+        count: 0,
+      },
+            links: {
+        first: '',
+        last: '',
+      }
     };
   }
-
-  return null;
 };
 
-export const mockCloneStatus = {
-  'f9133ec4-7a9e-4fd9-9a9f-9636b82b0a5d': {
-    options: {
-      ami: 'ami-0e778053cd490ad21',
-      region: 'us-west-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
-  '48fce414-0cc0-4a16-8645-e3f0edec3212': {
-    options: {
-      ami: 'ami-9f0asd1tlk2142124',
-      region: 'us-west-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
-  '0169538e-515c-477e-b934-f12783939313': {
-    options: {
-      ami: 'ami-9fdskj12fdsak1211',
-      region: 'us-west-2',
-    },
-    status: 'failure',
-    type: 'aws',
-  },
-  '4a851db1-919f-43ca-a7ef-dd209877a77e': {
-    options: {
-      ami: 'ami-9fdskj12fdsak1211',
-      region: 'eu-central-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
+export const mockCloneStatus = (cloneId: string): UploadStatus => {
+  if (cloneId === 'f9133ec4-7a9e-4fd9-9a9f-9636b82b0a5d') {
+    return ({
+      options: {
+        ami: 'ami-0e778053cd490ad21',
+        region: 'us-west-1',
+      },
+      status: 'success',
+      type: 'aws',
+    })
+  } else if (cloneId === '48fce414-0cc0-4a16-8645-e3f0edec3212') {
+    return ({
+      options: {
+        ami: 'ami-9f0asd1tlk2142124',
+        region: 'us-west-1',
+      },
+      status: 'success',
+      type: 'aws',
+    })
+  } else if (cloneId === '0169538e-515c-477e-b934-f12783939313') {
+    return ({
+      options: {
+        ami: 'ami-9fdskj12fdsak1211',
+        region: 'us-west-2',
+      },
+      status: 'failure',
+      type: 'aws',
+    })
+  } else {
+    // uuid: '4a851db1-919f-43ca-a7ef-dd209877a77e'
```

Comment:
This is a bit of a personal preference, but I feel like the long if/else if/else conditionals in this file are a bit cumbersome. It's very verbose and there are so many returns to keep track of.

It's also a bit of a code smell for me to keep track of which UUID we are returning in a comment as we do here in the else statement... we have to maintain the comment, and there is no guarantee that the comment will be correct in the future.

I would prefer to move back to use an object to keep track of everything, and just have a single return statement at the end of the function similar to `return statuses[uuid]`... but at the very least, I would like to move this UUID into its own else if and do something different with the else statement (we could consider some error handling but it might be overkill).